indico/modules/events/papers/templates/contributions.html

```diff
@@ -0,0 +1,83 @@
+{% from 'events/contributions/display/_paper_reviewing.html' import render_paper_reviewing %}
+
+{% macro render_paper_section(contribution, show_paper=True) %}
+    {% set state_mapping = {'submitted': '',
+                            'accepted': 'accept',
+                            'rejected': 'danger',
+                            'to_be_corrected': 'warning'} %}
+    {% if contribution.accepted_paper_revision and show_paper %}
+        <h3 class="icon-file">Paper</h3>
```

Comment:
Since you are re-working this page, I'd suggest that you refactor a bit the template to respect our template rules: https://github.com/indico/indico/wiki/Template-layout-(WIP)